src/effects/effectchainslot.cpp

```diff
@@ -275,6 +274,10 @@ QString EffectChainSlot::id() const {
     return m_id;
 }
 
+QString EffectChainSlot::group() const {
```

Comment:
Is this function necessary?

Reply:
I am using this function to fetch the formatted group name of the effect chain slot. Otherwise, I am constructing the Effect Chain Slot by passing only the chain slot number or group.

Reply:
okay